app/models/user.rb

```diff
@@ -91,8 +91,12 @@ module Permissions
   ####################
   # Class method(s)  #
   ####################
-  def self.search_by_email(email)
-    User.where('lower(email) like ?', "#{email}%")
+  def self.search_by_email_or_real_name(email, real_name)
```

Comment:
This would be clearer if instead of a single method that works differently depending on the parameters, we used two different methods. Since this only works for one or the other type of input, there's nothing we need a combined method for.